src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1136,12 +1138,22 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				language = KernelsLanguage.Python;
 			} else if (language.toLowerCase() === 'c#') {
 				language = KernelsLanguage.CSharp;
+			} else if (language.toLowerCase() === 'f#') {
```

Comment:
No error or anything if it isn't a language we recognize?

Reply:
This part is for translating the language into it's language registry equivalent. VS Code does similar translations to monaco editor equivalents.

https://github.com/microsoft/vscode/blob/main/extensions/ipynb/src/deserializers.ts#L10